src/blockstore/table.go

```diff
@@ -0,0 +1,234 @@
+package blockstore
+
+// This file implements a table manager for
+// key, value store in memory. Including a
+// LRU approximation cache
+
```

Comment:
What will this table hold? Where does this table fit into the larger design? I would love more comments at the top of this file explaining exactly how this table fits in.